dbc_extract3/dbc/generator.py

```diff
@@ -1993,6 +1993,9 @@ def __init__(self, options, data_store):
         if self._options.build >= dbc.WowVersion(8, 3, 0, 0):
             self._dbc.append('ItemBonus')
 
+        if self._options.build >= dbc.WowVersion(9, 0, 0, 0):
+            self._dbc.append('SpellTargetRestrictions')
+
```

Comment:
If you go by the new data model stuff (see comment below where you add the field to spell_data_t record output), explicitly instructing the generator to load db2 files is unnecessary.